supporting_scripts/hyperion/consistency-check-benchmark/exercises.py

```
@@ -68,25 +66,131 @@ def clone_pecv_bench(pecv_bench_dir: str) -> None:
+
+    After cloning/pulling, merges the dataset folders into the corresponding course folders
+    inside pecv_bench/data/{DATASET_VERSION}/QCSL25/.
+
+    :param str pecv_bench_dataset_dir: The directory where the dataset repository should be cloned
+    :raises SystemExit: if the git command fails
+    """
+    if os.path.exists(pecv_bench_dataset_dir):
+        logging.info(f"Directory {pecv_bench_dataset_dir} already exists. Pulling latest changes.")
+        try:
+            subprocess.run(
+                ["git", "reset", "--hard", "HEAD"],
```

Ruff (0.15.4)

[error] Starting a process with a partial executable path

(S607)